<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommentClass</class>
 <widget class="QDialog" name="CommentClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Comment</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#CommentClass {	
	font-family:&quot;微软雅黑&quot;;	
	background-color: rgb(255, 255, 255);
}

QLabel#iconLabel{
	background-image:url(:/QtWorkTest/Resources/commentDialogIcon.png);
    background-repeat: no-repeat;
    background-color: transparent;
    background-position:center;
	
}

QLabel#labelTop {
	background-repeat: no-repeat;
	background-color: transparent;
	background-position:center;
	font-family: Microsoft YaHei;
	font-size: 18px;
	color: #686868;
	line-height: 18px;
	padding-left: 10px;
	padding-top: 4px;
	padding-bottom: 10px;
}


QLabel#shadowLabel{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 rgb(218,218,218), stop: 1 white);
	margin-left: 2px;
}

QListWidget#listWidget {
	border: 0;
}

QTextEdit#textEdit {
	border: 0;
}

QPushButton#expressionButton {
	background-image: url(:/res/images/expression.png);
	background-position:center;
	background-repeat: no-repeat;
	background-color: transparent;
	background-size:cover;
}

QListWidget#listWidget{
	padding-left: 2px;
}
QListView#listWidget::item:selected {
	border: 0px;
}
QListView#listWidget::item:selected:!active {
	background: #FFFFFF;
}

QListView#listWidget::item:selected:active {
	background: #FFFFFF;
}
QListView#listWidget::item:hover{
	background: #FFFFFF;
}

QScrollBar{
    width:10px;
    border-radius:5px;
}
QScrollBar::handle{
	background:lightgray; border:2px ; border-radius:5px;
}
QScrollBar::handle:hover{
	background:gray; 
}
QScrollBar::handle:pressed{
    background:rgb(83,82,82);
}

QToolButton#minButton {
    background-image:url(:/QtWorkTest/Resources/mini_normal.png);
    background-repeat: no-repeat;
    background-color: transparent;
    background-position:center;
}
QToolButton#minButton:hover {
    background-image: url(:/QtWorkTest/Resources/mini_hover.png);
    background-repeat: no-repeat;
    background-color: transparent;
    background-position:center;
}

QToolButton#closeButton {
    background-image:url(:/QtWorkTest/Resources/close_normal.png);
    background-repeat: no-repeat;
    background-color: transparent;
    background-position:center;
}
QToolButton#closeButton:hover {
    background-image: url(:/QtWorkTest/Resources/close_hover.png);
    background-repeat: no-repeat;
    background-color: transparent;
    background-position:center;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>14</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>115</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="iconLabel">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTop">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>评论区</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="minButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="closeButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>5</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="shadowLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>7</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>7</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QListWidget" name="listWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="selectionRectVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="expressionButton">
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="sendButton">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/res/images/send.png);
background-color: transparent;
background-repeat: no-repeat;
background-position:center;
background-size:contain;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QTextEdit" name="textEdit">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>90</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtWorkTest.qrc"/>
 </resources>
 <connections/>
</ui>
